openid-4-verifiable-presentations-1_0.md

```diff
@@ -320,6 +320,24 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`attachments`:
+: OPTIONAL. Array of objects, where each object includes an extra resource that needs to be passed to the Wallet. These resources are intended to provide additional context to the Wallet or the End-User. The Wallet MUST ignore any unrecognized attachment formats. Each object contains the following parameters:
```

Comment:
As far as I understand, the intention of this parameter is to convey attestations/assertions/certificates confirming attributes of the Relying Party. Given this assumption, I find the name of the parameter and the description rather generic. Why is that?
I would rather expect something along the lines of "rp_credentials" or "client_credentials" or "client_attestations" or even "vp_token" since for me the difference between the `vp_token` response parameter and this parameter is just the direction and the entity the credential (presentations) are related to.

Reply:
I am fine renaming it. Joseph told me that we should limit it only for the purpose of credential requests and not to attach something like an ocsp response that can be used to validate the x509 that was used to sign the request.

`client_attestations` sounds better than `vp_token` because this name is already used (to have a better separation)